### DISCUSSION TITLE
Move icon themes to a GitHub repository

### DISCUSSION THREAD

**Discussion by cyrneko** [PrismLauncher/PrismLauncher#2233] [Ideas]
If you've wanted to install an icon theme in PrismLauncher in the past, you'll have come across this section of the Wiki:

![Screenshot from the website: Icons packs only on Discord for now: Fluent-Icons; Twemoji-Icons](https://media.wetdry.world/media_attachments/files/112/135/572/015/082/123/original/988e431d73a0c2d4.png)
> See also, the corresponding post: https://wetdry.world/@cyrus/112135575890769706

These are direct links to where you can find the icon themes **on Discord.** You cannot get them without either you or somebody else getting them from Discord first.

Whilst Icon-Packs are quite low-priority, I think moving them off of Discord onto GitHub has a couple of advantages:
- Decouples the download from any accounts (You need a Discord account to join the server and get the files)
- Makes the distribution of Icon Themes more open
- Is publically viewable, and indexable by search engines
- Easier to contribute[^1]
- Removes dependency on Discord[^2]

I'd love to assist in any way in making a move for the icon-themes to GitHub, for any reason, including the existing contributors and maintainers not having the time to take on this rather low-priority task at the moment.

[^1]: Right now, you have to (from my understanding) share your icon theme on Discord, and ask for someone with access to the site to merge a change that gives it a Discord-permalink on the Website.
[^2]: This is important because Discord is not a storage solution - Files or even the entire service could be gone at any moment, making these themes unavailable. This also includes the file being removed by Discord to save space.

**Comment by TheKodeToad** [comment]
See https://github.com/PrismLauncher/Themes/pull/19

**Comment by Scrumplex** [comment]
Done! We now have the two existing icon themes on GitHub here: https://github.com/PrismLauncher/Themes

And linked on the website here: https://prismlauncher.org/wiki/getting-started/change-themes/
